signac/contrib/project.py

```diff
@@ -336,7 +336,10 @@ def stores(self):
 
     @property
     def data(self):
-        """The data associated with this project.
+        """Use to read and write data associated with a `job`. It 
+        should be used for large array-like data, which can't be stored efficiently 
```

Comment:
To keep consistent with other doc strings:
```suggestion
"""The data associated with this project. 
This property should be used to read ...
```